products/red-hat-jboss-eap.md

```diff
@@ -50,7 +50,7 @@ releases:
     eoes: 2019-11-30
     latest: "5.2.0"
     latestReleaseDate: 2013-01-23 # unknown, date from https://access.redhat.com/documentation/en-us/jboss_enterprise_application_platform/5/html/release_notes_5.2.0/appe-release_notes_5.2-revision_history
-    link: https://access.redhat.com/documentation/en-us/jboss_enterprise_application_platform/5/html/release_notes_5.2.0
+    link: https://docs.redhat.com/documentation/red_hat_jboss_enterprise_application_platform/5/html/richfaces_developer_guide/index
```

Comment:
```suggestion
    link: https://web.archive.org/web/20230821184117/https://access.redhat.com/documentation/en-us/jboss_enterprise_application_platform/5/html/release_notes_5.2.0/index
```

https://docs.redhat.com/documentation/red_hat_jboss_enterprise_application_platform/5/html/richfaces_developer_guide/index is not the release notes for jboss AS 5.2.